tools/tools.py

```diff
@@ -167,9 +167,16 @@ def check(msg):
         
         
         await jobChannel.send(embed=randomize_colour(embed))
-        
-        
-        
-        
-        
 
+    @commands.command()
+    async def members(self, ctx, rolename):
+        role = discord.utils.get(ctx.message.guild.roles, name=rolename)
```

Comment:
This works totally fine but `ctx.guild` is a thing.

Reply:
Pretty sure I have to specify the guild. But if the master says so, so be it.